stars/game.py

```diff
@@ -108,7 +113,10 @@ def __init__(self, x=50, y=50, z=50, num_systems=10, system_names=None, races=[]
                 for p in self.players:
                     p.add_intel(b, {'location': b.location, 'size': b.radius})
             self._scan([])
-            self._call(self.players, 'update_stats')
+            if self.hundreth == 0:
```

Comment:
If we are only calculating score once, shouldn't that be the last hundreth not the first?

Reply:
this is the call to calculate score at the generation of the game